_documentation/general/beta.md

```diff
@@ -1,25 +1,11 @@
 ---
-title: Beta testing
+title: documentation.categories.general.beta
 layout: doc
 level: "2"
 group: "general"
 icon: "fas fa-vial"
 ---
 
-# Beta testing
+# {% t {{ page.title }} %}
 
-AntennaPod has many users and we don't want them to run into trouble when we add a new feature. It's important that we have a significant group test our app, so that we know all possible combinations of phones, Android versions and use cases work as expected.
-
-Minor issues often crop up after major updates, and they're only discovered via bug reports by users in these early stages. There is only one main developer who is working on this in his spare time. He only has a couple of devices to test things on, whilst behaviour can vary from device to device.
-
-## Google Play
-Every new version of AntennaPod is sent to our Beta Testers via the Google Play store, before it is shipped to the general public. It allows us to test updates on many different devices and Android versions, so we can catch bugs and user experience issues on time. If you have a critical eye and can explain what's going wrong, your help is very welcome! In addition, if you see something you don't like you're more likely to get it changed to match what you do want.
-
-So you want to have the latest and greatest? That's certainly possible! Simply visit the AntennaPod listing in the Play store and go to "Become a beta tester" at the bottom of the page. Or, if you're at your computer, you can go to [this url](https://play.google.com/apps/testing/de.danoeh.antennapod).
-
-<img src="/assets/images/documentation/project-beta.png" style="width:400px;" alt="Become a beta tester - confirmation window in Google Play">
-
-## F-Droid
-We do not currently offer beta versions on the F-Droid store.
-
-This is because F-Droid [does not](https://gitlab.com/fdroid/fdroidserver/-/issues/161) automatically recognise beta versions. Adding beta releases to F-Droid would require a manual intervention for each beta version. Also, if something important must be fixed, an update needs to be rolled out instantly. Unfortunately this cannot be guaranteed.
+{% capture content %}{% tf documentation/{{ page.group }}/{{ page.slug }}.md %}{% endcapture %}{{content | remove: "<!-- mdpo-disable -->" | remove: "<!-- mdpo-enable -->" | remove: "<!-- mdpo-disable-next-line -->" | remove: "<!-- mdpo-enable-next-line -->" }}
```

Comment:
Huh, why is this necessary here? (alternatively, why only here and not on all pages?)

Reply:
I just included the same code block stripping all mdpo tags from the pages where I used the image include. But having it on all pages is indeed more efficient. Where would you suggest to add it?

Reply:
Hmm, is it just to clean up the source code of the website or is it something that is visible to website users? If it is not visible to users, I would prefer using the shorter (easier to read/maintain) version.

Reply:
As an HTML comment it's only visible if you inspect the site code. Still, it's a bit weird. But I can actually move it to the main layouts – much simpler to maintain :)
Also should update the `lang` while at it.